### DISCUSSION TITLE
self signed certificate in certificate chain error

### DISCUSSION THREAD

**Discussion by Kathund** [community/community#35624] [Copilot Conversations] [answered]
When hovering over the github copilot icon it says `Copilot activation failed`

When clicked on it shows a dialog box saying `GitHub Copilot could not connect to server. Extension activation failed: "self signed certificate in certificate chain"`

When clicking on "Show output log" nothing happends and when going to the output it says `[ERROR] [default] [2022-10-10T01:05:29.013Z] GitHub Copilot could not connect to server. Extension activation failed: "self signed certificate in certificate chain"`

Anyone know what could be stopped copilot from connecting?

**Comment by fullstop125** [comment]
hi @Kathund  One way to try and solve this is to restart vs code and reset your browser to sign in again ...if this does not work you could try these useful steps:
run this code  : 

`git config --global http.sslVerify false`

after running this give it a try if it still doesn't work then you can try this article

https://mattferderer.com/fix-git-self-signed-certificate-in-certificate-chain-on-windows

**Reply by Kathund** [reply]
I have done everything in the linked article and I can't get it to work still any other ideas?

**Reply by Kathund** [reply, marked as answer]
[https://stackoverflow.com/questions/71367058/self-signed-certificate-in-certificate-chain-on-github-copilot](https://stackoverflow.com/questions/71367058/self-signed-certificate-in-certificate-chain-on-github-copilot) Found this online what explains and talks about an extenstion called [win-ca](https://marketplace.visualstudio.com/items?itemName=ukoloff.win-ca&ssr=false#overview) and it will sign all the certificates it self. all I had to do was install it and set the mode to **append** mode. Thanks for the help tho

**Reply by fullstop125** [reply]
awesome I am glad you got it working now
